web/src/app/main/NewApp.js

```diff
@@ -83,6 +84,7 @@ export default class App extends React.PureComponent {
             data-cy='app-bar'
             data-cy-format={cyFormat}
           >
+            <SkipToContentLink />
```

Comment:
I'm a little confused as to how this is being rendered. I don't see the link above the sidebar when in normal or screen reader mode. Can you touch on how to get this functionality showing?

Reply:
It only appears when it's focused, so if you're a keyboard-only user and you navigate the page using mostly the tab key, it allows you to skip over the repetitive content quickly.